flow/compositor_context.cc

```diff
@@ -130,6 +129,12 @@ RasterStatus CompositorContext::ScopedFrame::Raster(
           ? frame_damage->ComputeClipRect(layer_tree, !ignore_raster_cache)
           : std::nullopt;
 
+  if (aiks_context_ &&
+      !ShouldPerformPartialRepaint(clip_rect, layer_tree.frame_size())) {
+    clip_rect = std::nullopt;
+    frame_damage->Reset();
```

Comment:
This needs a null check for `frame_damage`, otherwise crashes with `FLTDisablePartialRepaint`.